MSAL/src/MSALAccount.m

```diff
@@ -184,4 +203,46 @@ - (BOOL)isEqualToUser:(MSALAccount *)user
     return result;
 }
 
+#pragma mark - Tenant profiles
+
+- (NSArray<MSALTenantProfile *> *)loadAllTenantProfiles:(MSALPublicClientApplication *)application error:(NSError **)error
```

Comment:
Do we need to require public client application? It seems weird to accept MSALPublicClientApplication inside MSALAccount.

Reply:
Changed. Now it becomes a readonly property :)